src/ui/profile_preferences.ui

```diff
@@ -3479,11 +3482,44 @@ you can use it but there could be issues with aligning columns of text</string>
           </item>
           <item row="5" column="0">
            <widget class="QCheckBox" name="checkBox_debugShowAllCodepointProblems">
+            <property name="toolTip">
+             <string>&lt;p&gt;If checked this will cause all problem Unicode codepoints to be reported in the debug output as they occur; if cleared then each different one will only be reported once and summarized in as a table when the console in which they occurred is finally destroyed (when the profile is closed).&lt;/p&gt;</string>
+            </property>
             <property name="text">
              <string>Report all Codepoint problems immediately</string>
             </property>
+           </widget>
+          </item>
+          <item row="6" column="0">
+           <widget class="QLabel" name="label_networkPackerTimeout">
+            <property name="text">
+             <string>Network packet timeout:</string>
```

Comment:
This text is great for an engineer who knows the context, but not so much for a player who doesn't. How about `Wait up to [   ]ms for more text to arrive`?